src/writeData/components/WriteDataItem.tsx

```diff
@@ -60,34 +41,15 @@ const WriteDataItem: FC<Props> = ({
     )
   }
 
-  if (onClick) {
-    return (
-      <SquareGrid.Card key={id}>
-        <SelectableCard
-          id={id}
-          formName="load-data-cards"
-          label={name}
-          selected={selected}
-          onClick={onClick}
-          testID={testID}
-          fontSize={ComponentSize.ExtraSmall}
-          className="write-data--item"
-        >
-          <div className="write-data--item-thumb">{thumb}</div>
-        </SelectableCard>
-      </SquareGrid.Card>
-    )
-  }
-
   return (
     <SquareGrid.Card key={id}>
       <SelectableCard
         id={id}
         formName="load-data-cards"
         label={name}
-        testID={`load-data-item ${id}`}
-        selected={false}
-        onClick={handleClick}
+        selected={selected}
+        onClick={onClick}
```

Comment:
Thanks for fixing this so that there's only one way to use it.

`onClick` is no longer optional. It should be updated in the interface to be required or we need a fallback here.